src/NLog/Targets/TraceTarget.cs

```diff
@@ -62,6 +63,12 @@ namespace NLog.Targets
     public sealed class TraceTarget : TargetWithLayout
     {
         /// <summary>
+        /// Always use <see cref="Trace.WriteLine(string)"/> independent of <see cref="LogLevel"/>
+        /// </summary>
+        [DefaultValue(false)]
+        public bool WriteLine { get; set; }
```

Comment:
- https://github.com/NLog/NLog/issues/1968: uses "prepend"
- https://github.com/NLog/NLog/issues/2234 uses "prefix"

I know this is depends of the listener, but still I prefer a name with prepend or prefix in it. What about PrependLogLevel?